Client/Frontend/Browser/BrowserTrayAnimators.swift

```diff
@@ -94,8 +94,29 @@ private extension TrayToBrowserAnimator {
             bvc.topToolbar.isTransitioning = false
             bvc.updateTabsBarVisibility()
             transitionContext.completeTransition(true)
+            
+            self.WKWebViewPDFNotRenderingBugFix(for: bvc)
```

Comment:
Can we do this only if a PDF file is opened, any easy way to check it?

Reply:
Done. You are right. There is a better way :) I found that each tab has a `mimeType` and I used that to determine if we are viewing a PDF. Also optimized it to not check every tab because it only ever happens to the selected tab.